src/typecode/contenttype.py

```diff
@@ -46,7 +46,9 @@
 
 from typecode import magic2
 from typecode import entropy
-
+#importing the archive.py from extract code
```

Comment:
Why this comment? I think that this is obvious what you are doing, comment not needed. Also we like imports to be sorted. So extractcode comes before typecode